test/integration/api-support/test/index.test.js

```diff
@@ -285,6 +286,17 @@ function runTests (serverless = false) {
     }
   })
 
+  if (!serverless) {
```

Comment:
What does this have to do with the serverless target?

Reply:
`next export` can only be used with `server`

Reply:
We should note that as a comment